### PR TITLE
IDEX-2989, IDEX-3010: Wrong behaviours with render package nodes and …

### PR DESCRIPTION
**Pull request by vzhukovskii**
…fix related with open file action
@vparfonov

### CODE REVIEW

File: plugin-java/che-plugin-java-ext-java/src/main/java/org/eclipse/che/ide/ext/java/client/project/node/SourceFolderNode.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import org.eclipse.che.ide.ext.java.shared.ContentRoot;
+import org.eclipse.che.ide.project.node.FolderReferenceNode;
+import org.eclipse.che.ide.project.node.resource.ItemReferenceProcessor;
+import org.eclipse.che.ide.ui.smartTree.presentation.NodePresentation;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+/**
+ * @author Vlad Zhukovskiy
```
Comment (skabashnyuk) [inline plugin-java/che-plugin-java-ext-java/src/main/java/org/eclipse/che/ide/ext/java/client/project/node/SourceFolderNode.java]: javadoc

**Comment by skabashnyuk** [comment]
other ok. What about @vparfonov ?

**Comment by vparfonov** [comment]
ok

**Comment by vzhukovskii** [comment]
merged with squash https://github.com/codenvy/che-plugins/commit/fe7647f65f16e4e690f6c368bde0aa003616f3c3

**Pull request closed without merging**